.overcommit.yml

```diff
@@ -0,0 +1,37 @@
+# Use this file to configure the Overcommit hooks you wish to use. This will
+# extend the default configuration defined in:
+# https://github.com/brigade/overcommit/blob/master/config/default.yml
+#
+# At the topmost level of this YAML file is a key representing type of hook
+# being run (e.g. pre-commit, commit-msg, etc.). Within each type you can
+# customize each hook, such as whether to only run it on certain files (via
+# `include`), whether to only display output if it fails (via `quiet`), etc.
+#
+# For a complete list of hooks, see:
+# https://github.com/brigade/overcommit/tree/master/lib/overcommit/hook
+#
+# For a complete list of options that you can use to customize hooks, see:
+# https://github.com/brigade/overcommit#configuration
+#
+# Uncomment the following lines to make the configuration take effect.
+
+PreCommit:
+  RuboCop:
+    enabled: true
+    on_warn: fail # Treat all warnings as failures
+
+  TrailingWhitespace:
+    enabled: true
+
+  RubyLint:
+    enabled: true
+    command: ['bundle', 'exec', 'ruby-lint']
+
+
+
+PostCheckout:
```

Comment:
@Nimmyjv What all are the default postcheckout hooks. Can you please comment here?

Its good to keep just bare minimum

Reply:
BowerInstall
BundleInstall
ComposerInstall
IndexTags
NpmInstall
SubmoduleStatus
YarnInstall